modules/create.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Kindly remove the eslint disable line, it is always better to fix all issues in the code.